package org.yschwartz.try_utils.exception;

public class ExceptionA extends RuntimeException {
    public String getMessageA() {
        return "Exception A";
    }
}
